retrieve-digests/pkg/retrieve_digests.go

```diff
@@ -0,0 +1,56 @@
+package pkg
+
+import (
+	"context"
+	"strings"
+)
+
+const (
+	registrySuseDe  = "registry.suse.de"
+	registrySuseCom = "registry.suse.com"
+	HeaderImage     = "Image"
+	HeaderTag       = "Tag"
+	HeaderDigest    = "Digest"
+	outputFileDe    = "suse_de.csv"
+	outputFileCom   = "suse_com.csv"
+)
+
+// RunRetrieveDigests contains the core logic for processing repositories
+func RunRetrieveDigests(project string) (int, int, error) {
+	projectURI := strings.ReplaceAll(strings.ToLower(project), ":", "/")
+
+	// Output files
+	outputFileDe := "suse_de.csv"
```

Comment:
There's a collision name between const and variable... so my recommentation is something like this:
Const with uppercase like (OUTPUT_FILE_DE = "suse_de.csv" and here:
remove the variable because you've got in the const so you can use it inside NewCSVWriter (OUTPUT_FILE_DE)